src/main/java/core/basesyntax/service/TransactionStrategy.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax.service;
+
+import core.basesyntax.FruitTransaction;
+
+public class TransactionStrategy {
+    public OperationHandler getShopService(FruitTransaction fruit) {
```

Comment:
still, bad method name
```suggestion
    public OperationHandler getOperationHandler(FruitTransaction fruit) {
```